pm/sendermonitor.go

```diff
@@ -422,7 +424,7 @@ func (sm *LocalSenderMonitor) redeemWinningTicket(ticket *SignedTicket) (*types.
 		if monitor.Enabled {
 			monitor.TicketRedemptionError()
 		}
-		return nil, err
+		return nil, errTicketRevert
```

Comment:
Returning `errTicketRevert` here obfuscates the error returned by `CheckTx` which also includes the hash of the reverted tx. As a result, we lose access to that information. Could we continue passing the error from `CheckTx` here and having a different check for a reverted tx error?